src/briefcase/integrations/android_sdk.py

```diff
@@ -420,21 +423,116 @@ def verify_emulator(self):
 
             return
 
-        self.command.logger.info("Downloading the Android emulator and system image...")
+        self.command.logger.info("Downloading the Android emulator...")
         try:
             self.command.subprocess.run(
                 [
                     os.fsdecode(self.sdkmanager_path),
                     "platform-tools",
                     "emulator",
-                    f"system-images;android-31;default;{self.emulator_abi}",
                 ],
                 env=self.env,
                 check=True,
             )
         except subprocess.CalledProcessError as e:
             raise BriefcaseCommandError(
-                "Error while installing Android emulator and system image."
+                "Error while installing Android emulator."
+            ) from e
+
+    def verify_avd(self, avd):
+        """Verify that the AVD has the necessary system components to launch.
+
+        :param avd: The AVD name to verify.
+        """
+        # Read the AVD configuration to retrieve the system image.
+        # This is stored in the AVD configuration file with the key:
+        #   image.sysdir.1=system-images/android-31/default/arm64-v8a/
+        try:
+            system_image_path = Path(self.avd_config(avd)["image.sysdir.1"])
+        except FileNotFoundError:
+            raise BriefcaseCommandError(f"Unable to read configuration of AVD @{avd}")
+        except KeyError:
+            self.command.logger.warning(
+                f"""
+*************************************************************************
+** WARNING: Unable to read AVD configuration                           **
+*************************************************************************
+
+    Briefcase was unable to read the configuration of the Android
+    emulator AVD {avd!r}.
+
+    Briefcase will proceeed assuming the emulator is correctly
+    configured. If you experience any problems running the emulator,
+    this is the likely cause of the problem.
+
+*************************************************************************
+"""
+            )
+            return
+
+        # Convert the path into a system image name, and verify it.
+        self.verify_system_image(";".join(system_image_path.parts))
```

Comment:
As noted above, this method should also ensure that the AVD's skin is installed.